.travis.yml

```diff
@@ -11,9 +11,12 @@ env:
   matrix:
     - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=file
     - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=debian
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
```

Comment:
how about allowing this to fail until it actually compiles under melodic...then make it not allowed to fail...otherwise even kinetic prs will fail...